ArduPlane/mode.h

```diff
@@ -946,6 +946,9 @@ class ModeAutoLand: public Mode
     AP_Int16 final_wp_dist;
     AP_Int16 landing_dir_off;
     AP_Int8  options;
+    AP_Int16 climb_before_turn_alt;
```

Comment:
It seems to be a standard to put _m on altitude variables in meters. This is in meters is it not?

Reply:
my understanding is that SI unit based variables do not need suffix, only those differing, ie _cm,_cd

Reply:
Should it be clarified in the style guide? https://ardupilot.org/dev/docs/style-guide.html

Reply:
Assuming my understanding is correct

Reply:
My understanding was that we did not add the suffix to the parameter name, but we did to the internal c++ variable.  Maybe I misunderstood.

Reply:
confirmed with Peter...we dont add suffix unless not SI or its confusing since there is a mismash of similar names in the module with differing units....will update the style guide to make it clear...